Atomic/backends/_docker.py

```diff
@@ -237,7 +242,27 @@ def start_container(self, name):
             raise ValueError("Unable to locate container '{}' in {} backend".format(name, self.backend))
         return self.d.start(name)
 
-    def stop_container(self, con_obj):
+    def stop_container(self, con_obj, **kwargs):
+        atomic = kwargs.get('atomic')
+        args = kwargs.get('args')
+        con_obj.stop_args = con_obj.get_label('stop')
+        if con_obj.stop_args:
+            try:
+                cmd = atomic.gen_cmd(con_obj.stop_args.split() + atomic.quote(args.args))
+            except TypeError:
+                cmd = atomic.gen_cmd(con_obj.stop_args + atomic.quote(args.args))
+            cmd = atomic.sub_env_strings(cmd)
+            atomic.display(cmd)
+            if args.display:
+                return 0
+            # There should be some error handling around this
+            # in case it fails.  And what should then be done?
+            return util.check_call(cmd, env=atomic.cmd_env())
+        elif args.display:
```

Comment:
Remove this, we don't display errors on other missing labels. (Run, Uninstall, Install).  No label showing up should tell the user that their is no label.